apps/client/src/app/pages/map/hooks/useHandleLogin.tsx

```diff
@@ -0,0 +1,11 @@
+function handleLogin(email: string, password: string) {
```

Comment:
please if you consider that this is a hook, the prefix of this function needs to be useHandleLogin, but this function doesn't require any other hook of react or ours, so this could be a service instead of a hook